src/client/public/locale/it/common.json

```
@@ -25,6 +25,7 @@
   "alls": "Tutti",
   "alteration": "Alterazione",
   "alternate_name": "Nome",
+  "analyze": "Analizzare",
```

```suggestion
  "analyze": "Analizzando",
```

🤷‍♀️  oder mal renato fragen (?)